doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,136 @@
+
+Building for the application and FLPR core
+******************************************
+
+This section outlines how to build and program for both the application and FLPR core, covering separate builds and sysbuild configurations.
+FLPR core supports two variants:
+
+* ``nrf54l15dk/nrf54l15/cpuflpr`` where FLPR runs from SRAM, which is the recommended method.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr`` snippet.
+
+* ``nrf54l15dk/nrf54l15/cpuflpr/xip`` where FLPR runs from RRAM.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr-xip`` snippet.
+
+Standard build
+--------------
+
+This subsection focuses on how to build an application using sysbuild.
+
+.. note::
+   Currently, the documentation does not cover specific instructions for building an application image that uses sysbuild to incorporate the FLPR core as a sub-image.
+   The only documented scenario is for building FLPR as the main image and the application as a sub-image.
+
+Complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: Using VPR Launcher
+
+      VPR launcher is a minimal sample that allows to start the FLPR core.
+      When using the VPR Launcher, the default configuration is employed for building with sysbuild targeting FLPR devices.
+      This process involves creating a minimal image that will be flashed onto the application core.
+      The primary function of this image is to transfer the FLPR code to the designated region (if necessary) and initiate the FLPR core.
+
+      To build and flash both images, run the following command:
+
+      .. code-block:: console
+
+         west build -p -b nrf54l15dk/nrf54l15/cpuflpr
+         west flash
+
+   .. group-tab:: Using application that supports multi-image builds
+
+      If your application involves creating custom images for both the application core and the FLPR core, make sure to disable the VPR Launcher.
+      You can do this by disabling the ``SB_CONFIG_VPR_LAUNCHER`` option when building for the FLPR target.
+      For more details, see :ref:`how to configure Kconfig <configuring_kconfig>`.
+
+      To build and flash both images, run the following command:
+
+      .. code-block:: console
+
+         west build -p -b nrf54l15dk/nrf54l15/cpuflpr
+         west flash
+
+Separate images
+---------------
+
+You can build and program application sample and the FLPR sample as separate images using the |nRFVSC| or command line.
+To use nRF Util, see `Programming application firmware on the nRF54L15 SoC`_.
+Depending on the selected method, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      .. note::
+
+         The Visual Studio Code extension currently offers experimental support for the nRF54L15's FLPR core.
+         Certain features, particularly debugging, may not function as expected.
```

Comment:
Looks very nice. Thank you for adding this note.
It has already been mentioned that 2 separate firmwares are built for 2 different cores, but the guide shows it like consecutive steps of building a single firmware. Can I ask for an explicit division between application core and flpr. For example:
`

      3.  Build the Application image. Set the following options:

         * Board target to ``nrf54l15dk/nrf54l15/cpuapp``.
         * Choose either ``nordic-flpr`` or ``nordic-flpr-xip`` snippet depending on the flpr image target
         * System build to :guilabel:`No sysbuild`.

         For more information, see :ref:`cmake_options`.
      
      #. Build the FLPR image. Set the following options:

         * Board target to ``nrf54l15dk/nrf54l15/cpuflpr`` (recommended) or ``nrf54l15dk/nrf54l15/cpuflpr/xip``.
         * System build to :guilabel:`No sysbuild`.

         For more information, see :ref:`cmake_options`.

`

Reply:
added